vms/platformvm/txs/executor/proposal_tx_executor.go

```diff
@@ -272,7 +282,10 @@ func (e *ProposalTxExecutor) AdvanceTimeTx(tx *txs.AdvanceTimeTx) error {
 
 	// Only allow timestamp to move forward as far as the time of next staker
 	// set change time
-	nextStakerChangeTime, err := GetNextStakerChangeTime(e.OnCommitState)
```

Comment:
See above: too many places where this is called already, hard to track if we merge latest avax changes again